_posts/henry_zhao/2019-03-06-Implement-Stardust.md

```diff
@@ -0,0 +1,287 @@
+---
+layout: post
+title: 'Accelerate data visualization with WebGL Platform Stardust'
+breadcrumb: true
+author: henry_zhao
+date: 2019-03-06
+categories: henry_zhao
+tags:
+  - Javascript
+  - WebGL
+  - Stardust
+teaser:
+  info: Data visualization with massive data could be fine in your browser
+  image: henry_zhao/stardust.png
+header:
+  version: small
+  title: Software Engineering Blog
+  image: header-logo-crop.png
+  icon: icon-blog
+---
+
+<image src="{{ site.urlimg }}/henry_zhao/future_world.jpg" />
+
+# Introduction
+
+Genomic data is huge and messy. In order to provide researchers a powerful data analysis&management platform, what we present data is the most popular data processing library, d3. It has a wealth of tools that can use svg to portray and manipulate data patterns to the greatest extent possible with different ranges and different forms of pie chart, histograms charts and human figures. For example:
```

Comment:
We might want a different word than `messy`, @rosibaj an idea?

Reply:
`complicated` ?

Reply:
disorganized? @Fgerthoffert @rosibaj